client/src/pages/ProfileListings/ProfileListings.tsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { useState, useEffect } from 'react';
+import { Grid, CssBaseline, TextField, InputAdornment, Typography, Button } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import DateRangeIcon from '@material-ui/icons/DateRange';
+import ProfilePreview from './ProfilePreview/ProfilePreview';
+import generateMockData from './generateMockData';
+import { defaultUsers } from './mockData';
+import useStyles from './useStyles';
+
+const ProfileListings = (): JSX.Element => {
+  const classes = useStyles();
+  const [currentUsers, setCurrentUsers] = useState<number>(6);
+  const [mockData, setMockData] = useState<any>([]);
```

Comment:
Please avoid using 'any' as a type, you can create a mockProfile interface instead :)

Reply:
The interface will be implemented when the FE is integrated with the BE. Currently the data is being pulled from https://randomuser.me/ and any interface implemented here will not be used when the FE is integrated.